src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.skia.cs

```diff
@@ -3148,6 +3149,34 @@ public async Task When_Ctrl_Delete_Undo_Redo()
 			Assert.AreEqual("world", SUT.Text);
 		}
 
+		[TestMethod]
+		public async Task When_FeatureConfiguration_Changes()
+		{
+			var useOverlay = FeatureConfiguration.TextBox.UseOverlayOnSkia;
+			using var _ = Disposable.Create(() => FeatureConfiguration.TextBox.UseOverlayOnSkia = useOverlay);
+
+			FeatureConfiguration.TextBox.UseOverlayOnSkia = true;
+
+			var SUT = new TextBox
+			{
+				Width = 150,
+				Text = "hello world"
+			};
+
+			await UITestHelper.Load(SUT);
+
+			SUT.Focus(FocusState.Programmatic);
+			await WindowHelper.WaitForIdle();
+
+			Assert.AreEqual(0, SUT.TextBoxView.DisplayBlock.Opacity);
+
+			FeatureConfiguration.TextBox.UseOverlayOnSkia = false;
```

Comment:
Would you need to have a reversed test when the overlay box is enabled by default in your other PR?

Reply:
I assume you mean disabled* by default? The test sets `UseOverlayOnSkia` to true and restores it afterwards, so it the default value shouldn't matter anyway.